Makefile

```diff
@@ -17,6 +17,11 @@ CHPL := chpl
 # We need to make the HDF5 API use the 1.10.x version for compatibility between 1.10 and 1.12
 CHPL_FLAGS += --ccflags="-DH5_USE_110_API"
 
+# Python
+ifdef ARKOUDA_PYTHON_SERVER
+CHPL_FLAGS += $(shell python3-config --includes) $(shell python3-config --ldflags) -lpython3.$(shell python -c "import sys; print(sys.version_info.minor)")
```

Comment:
A note that running `python3-config --ldflags` on mac ends with `'-framework CoreFoundation'` and chpl doesn't know how to handle that. It causes the make to error with `Unrecognized flag: '-f'`

This is prob not the best way to do this, but I modified the make file to remove this suffix. That made it so I could build the server
```
python_ld_flags=$(shell python3-config --ldflags)

# mac has '-framework CoreFoundation' and chpl doesn't know how to deal with it
# so we're gonna remove it
mac_framework_suffix="-framework CoreFoundation"

CHPL_FLAGS += $(shell python3-config --includes) ${python_ld_flags%"$mac_framework_suffix"}  -lpython3.$(shell python -c "import sys; print(sys.version_info.minor)")
```